src/Softeq.NetKit.Chat.Domain.Services/DomainServices/ChannelService.cs

```diff
@@ -168,15 +177,18 @@ public async Task<ChannelSummaryResponse> CreateDirectChannelAsync(CreateDirectC
             using (var transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
             {
                 await UnitOfWork.ChannelRepository.AddChannelAsync(newChannel);
+
```

Comment:
Did you merge latest develop into your branch? We got rid of this lines "using (var transactionScope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))" in develop and extracted it in the UnitOfWork.